### ISSUE TITLE
Share Logs

### ISSUE THREAD

**Issue by tarna** [Minehut/Meta#934]
### Feature request

**Checklist**
- [x] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [x] This is not a bug report, support request or plugin request
- [x] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [x] This issue does not contain any sensitive information 

**Platform**

Website

**Describe the feature**

Share button when inside a log file that will automatically upload it to pastebin or Minehut's own service which can be used to get a link to share the logs to others.

**How the feature is useful**

This is useful for anyone who is having issues with their server and want to share it with others so they can help troubleshoot. And instead of copying the logs, going to pastebin, generating a link, and sending it to someone. The user would just click the share button and it will automatically copy a link of the links to the user's clipboard in which they can share. These links can also be temporary if needed.

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by Tringlle** [comment]
Yeah, I know on Pterodactyl Panel, there is a _addon_ that adds this feature, it uploads to mcpaste, but idk if minehut wants to use their own service. This doesn't seem too hard, as they just use the already built download feature to send that file to the service
